lib/rules/no-empty-first-line/README.md

```diff
@@ -0,0 +1,31 @@
+# no-empty-first-line
+
+Disallow empty first lines.
+
+```css
+    \n
+    /** ↑
+     * This newline */
+    a { color: pink; }
+```
+
+Completely empty files are not considered violations. These are handled by the rule [no-empty-source](../no-empty-source/README.md).
```

Comment:
I think the following is more in line with the language in other rule READMEs (and sources aren't always files):

```
This rule ignores empty sources. Use the [`no-empty-source`](../no-empty-source/README.md) rule to disallow these.
```